lib/monitor.js

```diff
@@ -121,10 +121,15 @@ internals.Monitor.prototype.start = function (callback) {
         };
     };
 
+    var reportersIndex = 0;
+
     Items.serial(this.settings.reporters, function (item, next) {
 
         var reporter;
 
+        // Use hint of reporter index
+        var errorHint = self.settings.reporters.length > 1 ? '(' + reportersIndex++ + ')' : '';
```

Comment:
It does start at `0`, this assert proves as much: https://github.com/kevinmstephens/good/commit/16036d29c283bfe6723902f391b217dfa5fc5553#diff-f24923ecc00214a4a695c4c4b28d23cbR341

Would you like to remove the logic that omits the hint index in the case of only one reporter?